test/evaluation/evaluationClient.spec.ts

```diff
@@ -165,57 +172,57 @@ describe('evaluate', () => {
     infoResults = infoResults.filter(result => result.evaluator === 'MarkForSubmissionEvaluation');
     expect(infoResults.length).toEqual(1);
     expect(errorResults.length).toEqual(2);
-    expect(evaluationClient.verifiablePresentation.getVerifiableCredentials().length).toEqual(1);
-    expect(Object.keys(evaluationClient.verifiablePresentation.getVerifiableCredentials()[0]).length).toEqual(6);
+    expect(evaluationClient.verifiablePresentation.verifiableCredential.length).toEqual(1);
+    expect(Object.keys(evaluationClient.verifiablePresentation.verifiableCredential[0]).length).toEqual(8);
   });
 
 
   it('should return ok if limit_disclosure deletes the etc field', function() {
     const presentationDefinition: PresentationDefinition = getFile('./test/dif_pe_examples/pd/pd-simple-schema-age-predicate.json').presentation_definition;
-    const vpSimple = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
     const evaluationClient: EvaluationClient = new EvaluationClient();
     vpSimple.holder = HOLDER_DID;
-    evaluationClient.evaluate(presentationDefinition, new VP(vpSimple));
-    expect(evaluationClient.verifiablePresentation.getVerifiableCredentials()[0]['etc']).toEqual(undefined);
+    evaluationClient.evaluate(presentationDefinition, vpSimple);
+    expect(evaluationClient.verifiablePresentation.verifiableCredential[0]['etc']).toEqual(undefined);
   });
 
   it('should return error if limit_disclosure deletes the etc field', function() {
     const presentationDefinition: PresentationDefinition = getFile('./test/dif_pe_examples/pd/pd-simple-schema-age-predicate.json').presentation_definition;
-    const vpSimple = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
```

Comment:
we had to move these to TypeScript files from json files.

ps. all json example should be moved to ts.